app/public/Console/consoleWindow.js

```diff
@@ -0,0 +1,105 @@
+const CONSOLE = (() => {
+
+    const consoleWindowElement = document.getElementById("consoleWindow");
+    const lineHolderElement = document.createElement("div");
+    lineHolderElement.className = "lineHolder";
+    consoleWindowElement.appendChild(lineHolderElement);
+
+    const scrollElement = consoleWindowElement;
+
+    const snapToBottomTolerance = 50; //When within this many pixels from the bottom of the console, new messages will cause the scroll to snap to the bottom
```

Comment:
Nitpick but snapToBottomTolerance should probably be all caps since it's a constant